solidjs-tailwind/src/services/get-repo-pull-requests.js

```diff
@@ -121,21 +107,63 @@ const getRepoPullRequests = async ({
       authorization: `Bearer ${authStore.token}`,
     },
   };
-  const resp = await FetchApi(data);
 
-  const openPullRequests = parsePullRequests(
-    resp.data.repository?.openPullRequest
-  );
-  const closedPullRequests = parsePullRequests(
-    resp.data.repository?.closedPullRequest
-  );
+  let openPullRequests;
+  let closedPullRequests;
+  let milestones;
+  let labelMap;
```

Comment:
Since these values will change only when we re-do an API call I think it is better to use consts instead of lets. I'll share an example below on how to do this